### PR TITLE
Move Option Converter to new package/ project

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: Common/AssemblyInfo.fs [hunk 1]
```diff
@@ -34,8 +34,8 @@ open System.Runtime.InteropServices
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [<assembly: AssemblyVersion("1.0.*")>]
-[<assembly: AssemblyVersion("0.1.0.0")>]
-[<assembly: AssemblyFileVersion("0.1.0.0")>]
+[<assembly: AssemblyVersion("0.1.2.0")>]
```
Comment (jebbinSCL) [inline Common/AssemblyInfo.fs]: I made a mistake here skipping a version, but its already been uploaded to nuget. I've only just found that there is a staging server for nuget so you an check what is uploaded.

Comment (pallenSL) [inline Common/AssemblyInfo.fs]: This means we cannot reproduce that build.
Potentially serious on a real project.

Comment (jebbinSCL) [inline Common/AssemblyInfo.fs]: As in reproduce the missing version 0.1.1.0?

Comment (pallenSL) [inline Common/AssemblyInfo.fs]: Unless I have misunderstood, 0.1.2.0 is on NuGet but the codebase for it has not been checked in?

Comment (jebbinSCL) [inline Common/AssemblyInfo.fs]: Ah, no, Nuget is 0.1.2.0 and this pull request code is 0.1.2.0.

Comment (jebbinSCL) [inline Common/AssemblyInfo.fs]: That's why I said it was a mistake. I was unclear about the result of the package on nuget, so I was trying to test it. However, I believe I can upload the same version again to override it.

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**